etc/timescaledb/values.yaml

```diff
@@ -1,472 +1,80 @@
   ##
-  # loadBalancerSourceRanges:
-  # - 10.10.10.0/24
-
-## Start master and slave(s) pod(s) without limitations on shm memory.
-## By default docker and containerd (and possibly other container runtimes)
-## limit `/dev/shm` to `64M` (see e.g. the
-## [docker issue](https://github.com/docker-library/postgres/issues/416) and the
-## [containerd issue](https://github.com/containerd/containerd/issues/3654),
-## which could be not enough if PostgreSQL uses parallel workers heavily.
+  applicationName: grafana
+## @param postgresqlDataDir PostgreSQL data dir folder
 ##
-shmVolume:
-  ## Set `shmVolume.enabled` to `true` to mount a new tmpfs volume to remove
-  ## this limitation.
-  ##
-  enabled: true
-  ## Set to `true` to `chmod 777 /dev/shm` on a initContainer.
-  ## This option is ingored if `volumePermissions.enabled` is `false`
-  ##
-  chmod:
-    enabled: true
+postgresqlDataDir: /bitnami/postgresql/data
 
-## PostgreSQL data Persistent Volume Storage Class
-## If defined, storageClassName: <storageClass>
-## If set to "-", storageClassName: "", which disables dynamic provisioning
-## If undefined (the default) or set to null, no storageClassName spec is
-##   set, choosing the default provisioner.  (gp2 on AWS, standard on
-##   GKE, AWS & OpenStack)
+## @section PostgreSQL Primary parameters
 ##
-persistence:
-  enabled: true
-  ## A manually managed Persistent Volume and Claim
-  ## If defined, PVC must be created manually before volume will be bound
-  ## The value is evaluated as a template, so, for example, the name can depend on .Release or .Chart
-  ##
-  # existingClaim:
-
-  ## The path the volume will be mounted at, useful when using different
-  ## PostgreSQL images.
-  ##
-  mountPath: /bitnami/postgresql
-
-  ## The subdirectory of the volume to mount to, useful in dev environments
-  ## and one PV for multiple services.
+primary:
+  ## @param primary.extraEnvVars Array with extra environment variables to add to PostgreSQL Primary nodes
+  ## e.g:
+  ## extraEnvVars:
+  ##   - name: FOO
+  ##     value: "bar"
   ##
-  subPath: ""
-
-  # storageClass: "-"
-  accessModes:
-    - ReadWriteOnce
-  size: 8Gi
-  annotations: {}
-
-## updateStrategy for PostgreSQL StatefulSet and its slaves StatefulSets
-## ref: https://kubernetes.io/docs/concepts/workloads/controllers/statefulset/#update-strategies
-##
-updateStrategy:
-  type: RollingUpdate
-
-##
-## PostgreSQL Master parameters
-##
-master:
-  ## Node, affinity, tolerations, and priorityclass settings for pod assignment
-  ## ref: https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#nodeselector
-  ## ref: https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#affinity-and-anti-affinity
-  ## ref: https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#taints-and-tolerations-beta-feature
-  ## ref: https://kubernetes.io/docs/concepts/configuration/pod-priority-preemption
+  extraEnvVars:
+    - name: POSTGRESQL_USERNAME
```

Comment:
For us `POSTGRES_USER` works instead of `POSTGRESQL_USERNAME`.

Reply:
@blootsvoets I'm also getting error if I use `POSTGRESQL_USERNAME`.

Reply:
I recall getting another error if `POSTGRESQL_USERNAME` was not set in an initialisation script. I can just set both.

Reply:
Fixed in a8938e15fc74d8daba9d2825986f4081d0bf0bbf